### ISSUE TITLE
Simulation keeps running in the background

### ISSUE THREAD

**Issue by dev-zero** [CrayLabs/SmartSim-workshops#3]
I made the mistake of running the solution with an unmodified simulation, which causes the simulation to run forever (this might also be something document: how long the simulation should take).
The problem was that a `ctrl+c` did not kill that simulation but it kept running, ending the PBS allocation helped, but is cumbersome.

**Comment by al-rigazzi** [comment]
Yeah, this is strange because we usually capture those. There might be something going on with PBS
